tools/prometheus-builder/build.sh

```diff
@@ -24,6 +24,16 @@ fi
 
 git checkout pr-branch
 
+# Here, STORAGE is specified as an  environment variable of the prometheus-builder init container, 
+# where it will copy the key.yml file from the Prometheus directory to the volume section of the
+# emptyDir. This file will later be used by the data-downloader init container.
+if [ -f "$DIR/key.yml" ]; then
+    echo "INFO:: key.yml file is Present on $DIR/key.yml directory."
+    cp  "$DIR/key.yml" "$STORAGE/key.yml"
+else
+    echo "INFO:: key.yml File does not exist on $DIR/key.yml directory."
```

Comment:
We should add more context or remove this line. e.g. File does not exists so we won't download XXX